src/structuredbroadcast.jl

```diff
@@ -27,28 +31,46 @@ Broadcast.BroadcastStyle(::StructuredMatrixStyle{Diagonal}, ::StructuredMatrixSt
     StructuredMatrixStyle{LowerTriangular}()
 Broadcast.BroadcastStyle(::StructuredMatrixStyle{Diagonal}, ::StructuredMatrixStyle{<:Union{UpperTriangular,UnitUpperTriangular}}) =
     StructuredMatrixStyle{UpperTriangular}()
+Broadcast.BroadcastStyle(::StructuredMatrixStyle{Diagonal}, ::StructuredMatrixStyle{UpperHessenberg}) =
+    StructuredMatrixStyle{UpperHessenberg}()
```

Comment:
Just out of curiosity, where do you handle the case `Broadcast.BroadcastStyle(::StructuredMatrixStyle{UpperHessenberg}, ::StructuredMatrixStyle{UpperHessenberg})`? Like in `UH .+ UH`. Is there some fallback?